src/officehours_api/views.py

```diff
@@ -73,9 +77,24 @@ def check_change_permission(self, request, user):
         if user != request.user:
             self.permission_denied(request)
 
+    def verify_phone(self, request, user):
+         client = Client()
+         if user != request.user:
+             if User.phone_number != None:
```

Comment:
I'm not sure I understand these checks yet. For `user != request.user`, are we just checking to see if the user has been changed? I think we probably want to check specifically for if the phone number has changed and then run the verification. `User.phone_number` puzzles me, cause `User` is the model, not the model instance. We probably do need a check like this to make sure we're not passing `None` into this verification process.